config/cors.php

```diff
@@ -17,9 +17,9 @@
 
     'paths' => ['api/*', 'api', 'sanctum/csrf-cookie'],
 
-    'allowed_methods' => ['*'],
+    'allowed_methods' => ['POST', 'GET', 'OPTIONS', 'PUT', 'DELETE'],
```

Comment:
Do we really need this? When we put *, that means ['POST', 'GET', 'OPTIONS', 'PUT', 'DELETE'].

Reply:
Yes we need this for CORS, it common knowledge